scripts/setup-project.sh

```diff
@@ -5,7 +5,7 @@ set -e
 show_help() {
   echo "Usage: $0 <project_name>"
   echo "Available projects: stm32l4, mimxrt1060, same54xpro, " \
-      "same54xprov2, maaxboardrt, rx65ncloudkit, wfi32iot"
+      "same54xprov2, maaxboardrt, ck-rx65n, rx65ncloudkit, wfi32iot"
```

Comment:
Maybe we can add a bit here to disambiguate between the ck-rx65n and rx65ncloudkit. I don't think it would be immediately obvious which one belongs to which target. Something like:

rx65ncloudkit - Green PCB, WiFi module supported, no TSIP support
ck-rx65n - Blue PCB, Ethernet supported, TSIP support

Reply:
Done